client/src/views/Dashboard/GroupedBarChart/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+    BarChart,
+    Bar,
+    XAxis,
+    YAxis,
+    Tooltip,
+    Legend,
+    ResponsiveContainer,
+    LabelList,
+} from 'recharts';
+
+function CustomizedLabel(props) {
+    const { x, y, width, value } = props;
+
+    return (
+        <text
+            x={x + width / 2.75}
+            y={y}
+            dy={-8}
+            fontSize="12"
+        >
+            {value}
+        </text>
+    );
+}
+
+export default function GroupedBarChart(props) {
+    const {
+        data,
+    } = props;
+
+    if (data.values.length <= 0) {
```

Comment:
Import isNotDefined from togglecorp/fujs
```suggestion
    if (isNotDefined(data) || isNotDefined(data.values) || data.values.length <= 0) {
```

Reply:
okay !!